packages/font-icons-mdl2/src/fabric-icons-0.ts

```diff
@@ -1,10 +1,9 @@
   // Your use of the content in the files referenced here is subject to the terms of the license at https://aka.ms/fluentui-assets-license
 
 import {
```

Comment:
I'm pretty sure these are auto-generated by some tool, so that might need to be updated as well.

Reply:
the code to generate this is in an ODSP repo (the font site that Peter built) and we're very unlikely to regenerate the icons ever again, especially with the move to SVG and lack of investment in font-based.

Reply:
I think if we do generate again, by that time we'll have the lint rule running (along with isolatedModules) which will fail unless it's correct, so at least there are gates.